Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:20.11.1 AS builder
```

Comment:
this will do the build in a seperate stage so that the final image can be kept a bit smaller without including any of the extra dependancies that are needed at build time but not at run time